internal/step/stepper.go

```diff
@@ -76,12 +76,9 @@ func (s *Factory) DisplayLogHints(cloudProvider string, estimatedTime int) {
 		documentationLink += cloudProvider + `/quick-start/install/cli`
 	}
 
-	header := `
-##
-# Welcome to Kubefirst
-`
+	header := "\n Welcome to Kubefirst \n\n"
```

Comment:
Another newline we could probably remove

```suggestion
	header := "Welcome to Kubefirst \n\n"
```